posthog/queries/trends/util.py

```diff
@@ -142,3 +142,10 @@ def ensure_value_is_json_serializable(value: Any) -> Optional[float]:
             capture_exception(exception)
         return None
     return value
+
+
+def determine_aggregator(entity: Entity, team: Team) -> str:
```

Comment:
Nit: Could we make Team the first argument? Reasoning: it's the most 'generic' argument and a lot of functions accept teams - having it first creates consistency.

Reply:
I kind of agree, but I also wanted to maintain consistency with the ordering in `process_math()` and the call site – `_total_volume_query()` – so this order is intentional